app/services/notifications/ad-hoc-returns/initiate-session.service.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
```

Comment:
Was there a reason why we couldn't use `app/services/bill-runs/setup/initiate-session.service.js` to do this if we moved it to a more generic folder?